dev-manual/api/api-reference-storage-service.rst

```diff
@@ -540,13 +544,330 @@ Where payload.json contains::
+
+DSpace via SWORD2 API
+^^^^^^^^^^^^^^^^^^^^^
+
+DSpace via SWORD2 locations are currently supported only for AIP Storage locations.
+Basic fields for a local file system space are listed `above \
+<https://www.archivematica.org/en/docs/archivematica-1.15/dev-manual/api/api-reference-storage-service/#create-space>`_
+
+Request body parameters:
+
+===================     =======================================================================================
+
+``sd_iri``              URL of the service document. E.g. ``http://demo.dspace.org/swordv2/servicedocument,``
+                        where servicedocument is the handle to the community or collection being used
+                        for deposit.
+
+``user``                A username for the DSpace instance with sufficient permissions to permit
+                        authentication.
+
+``password``            The password for the Dspace user that you created.
+
+``metadata_policy``     Use to restrict access to the metadata bitstream. Must be specified as a list of
+                        objects in JSON, e.g. ``[{"action":"READ","groupId":"5","rpType":"TYPE_CUSTOM"}]``.
+
+``archive_format``      Select Archive format from dropdown. e.g. ``ZIP or 7z``
+
+===================     =======================================================================================
+
+DSpace via REST API
+^^^^^^^^^^^^^^^^^^^
+
+DSpace via REST API locations are supported for both AIP and DIP Storage locations.
+Basic fields for a local file system space are listed `above \
+<https://www.archivematica.org/en/docs/archivematica-1.15/dev-manual/api/api-reference-storage-service/#create-space>`_
+
+Request body parameters:
+
+======================    =====================================================================================
+
+``ds_rest_url``           URL to the "REST" webapp. E.g. ``http://localhost:8080/rest/``; for production
+                          systems, this address will be slightly different, such as:
+                          ``https://demo.dspace.org/rest/``.
+
+``ds_user``               A username for the DSpace instance with sufficient permissions to permit
+                          authentication.
+
+``ds_password``           The password for the Dspace user that you created.
+
+``ds_dip_collection``     UUID of default DSpace collection for the DIP to be deposited.
+
+``ds_aip_collection``     UUID of default DSpace collection for the AIP to be deposited.
+
+``as_url``                URL to the ArchiveSpace server. E.g.: ``http://sandbox.archivesspace.org/``.
+
+``as_user``               ArchivesSpace username to authenticate as.
+
+``as_password``           ArchivesSpace password to authenticate with.
+
+``as_repository``         Identifier of the default ArchivesSpace repository.
+
+``as_archival_object``    Identifier of the default ArchivesSpace archival object.
+
+``upload_to_tsm``         Boolean True or False. Default is False. will attempt to send the AIP to
```

Comment:
If the content type of this endpoint is `json`, these boolean values are incorrect.

```suggestion
``upload_to_tsm``         Boolean true or false. Default is false. will attempt to send the AIP to
```